src/site/content/en/blog/vr-comes-to-the-web/index.md

```diff
@@ -213,8 +221,8 @@ the information needed to render a single frame to the display. The
 Before doing anything else, I'm going to request the next animation frame. As
 previously stated, the timing of frames is determined by the user agent based on
 the underlying hardware. Requesting the next frame first ensures that if
-something during the callback throws an error I can ensure that the frame loop
-continues.
+the underlying hardware. Requesting the next frame first ensures that
```

Comment:
Insertion error here--I think because GH's multi-line comments don't let you do multi-line edits, so my bad.

Reply:
Fixed.